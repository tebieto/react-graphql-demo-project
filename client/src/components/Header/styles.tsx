import { AppBar, Toolbar } from '@material-ui/core';
import styled from 'styled-components';

export const HeaderContainer = styled(AppBar)`
  display: flex !important;
  padding: 15px 200px;
  @media screen and (max-width: 800px) {
    padding: 15px 10px;
    justify-content: space-between;
  }
  z-index: 1 !important;
`;

export const ToolbarContainer = styled(Toolbar)`
  display: flex;
  align-items: center;
  justify-content: space-between;
  z-index: 1;
`;
