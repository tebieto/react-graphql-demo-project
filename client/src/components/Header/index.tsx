import React from 'react';
import HeaderProfileSection from '../HeaderProfileSection';
import Logo from '../Logo';
import { HeaderContainer, ToolbarContainer } from './styles';

const Header = (): JSX.Element => {
  return (
    <HeaderContainer color="secondary" elevation={1} position="static">
      <ToolbarContainer>
        <Logo />
        <HeaderProfileSection />
      </ToolbarContainer>
    </HeaderContainer>
  );
};

export default Header;
