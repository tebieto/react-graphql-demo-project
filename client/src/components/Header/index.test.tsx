import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '.';

it('renders correctly', () => {
  const tree = shallow(<Header query="test" />);
  expect(toJson(tree)).toMatchSnapshot();
});
