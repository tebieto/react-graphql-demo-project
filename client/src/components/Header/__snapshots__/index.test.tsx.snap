// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Styled(WithStyles(ForwardRef(AppBar)))
  color="secondary"
  elevation={1}
  position="static"
>
  <Styled(WithStyles(ForwardRef(Toolbar)))>
    <Logo />
    <div
      className="search-input-container"
    >
      <CustomInput
        handleIconClick={[Function]}
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="Search"
        size="small"
        value=""
      />
    </div>
    <HeaderProfileSection />
  </Styled(WithStyles(ForwardRef(Toolbar)))>
</Styled(WithStyles(ForwardRef(AppBar)))>
`;
